plugin/data_masking/src/udf/udf_gen_blacklist.cc

```diff
@@ -13,23 +13,28 @@
    along with this program; if not, write to the Free Software Foundation,
    51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
 
+#include <my_global.h>
 #include "../../include/plugin.h"
 #include "../../include/udf/udf_utils.h"
 #include "../../include/udf/udf_utils_string.h"
 
 #include <algorithm>
 
 extern "C" {
-  bool gen_blacklist_init(UDF_INIT *initid, UDF_ARGS *args, char *message);
+  my_bool gen_blacklist_init(UDF_INIT *initid, UDF_ARGS *args, char *message);
```

Comment:
what happens with bool?
How did it work before? for example, std::snprintf() was used with bool ?

Reply:
Ok, thanks @ldonoso I think this change is fine for 5.7 only.
For 8.0, Please keep the bool as is.

Reply:
Yes, I will do. Please, note @satya-bodapati that this commit is just intended for 5.7.

Reply:
yes, luis.